composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewModel.kt

```diff
@@ -0,0 +1,69 @@
+package com.majotyler.hiittimer.presentation
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+class TimerViewModel : ViewModel() {
+
+    private val _loading = MutableStateFlow(false)
+    val loading = _loading.asStateFlow()
+
+    private val _exercises = MutableStateFlow(listOf("Lagartijas","gallitos", "dominadas"," sentadillas"))
+    val exercises = _exercises.asStateFlow()
+
+    fun onEvent(event: TimerViewEvent) {
+        when (event) {
+            is TimerViewEvent.ClickAdd -> {
+                viewModelScope.launch {
+                    ClickAdd()
+                }}
+            is TimerViewEvent.onDelete -> {
+                onDelete(event.item)
+                }
+
+    }
+    }
+
+    suspend fun ClickAdd() {
```

Comment:
Functions should always start with lower-case letters, so this should be `clickAdd()` or even better (in my quisquilloso opinion) `onClickedAdd()`